packages/@react-aria/tabs/docs/useTab.mdx

```diff
@@ -0,0 +1,162 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-aria/tabs';
+import {HeaderInfo, FunctionAPI, TypeContext, InterfaceType} from '@react-spectrum/docs';
+import packageData from '@react-aria/tabs/package.json';
+import Anatomy from './anatomy.svg';
+
+```jsx import
+import {useTab, useTabList, useTabPanel} from '@react-aria/tabs';
+import {useTabListState} from '@react-stately/tabs';
+```
+
+---
+category: Navigation
+keywords: [tabs, aria]
+---
+
+# useTabList
+
+<p>{docs.exports.useTab.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['useTabList', 'useTab', 'useTabPanel']}
+  sourceData={[
+    {type: 'W3C', url: 'https://www.w3.org/TR/wai-aria-practices/#tabpanel'}
+  ]} />
+
+## API
+
+<FunctionAPI function={docs.exports.useTabList} links={docs.links} />
+
+<FunctionAPI function={docs.exports.useTab} links={docs.links} />
+
+<FunctionAPI function={docs.exports.useTabPanel} links={docs.links} />
+
+## Features
+
+Tabs provide a list of tabs that a user can select from to change the panels contents. `useTab`, `useTabList`, and `useTabPanel` help implement these in an accessible way.
+
+* Support for mouse, touch, and keyboard interactions on tabs
+* Support for LTR and RTL keyboard navigation
+* Support for disabled tabs
+* Support for accessibly linking tabs and the associated tab panel
+
+
+## Anatomy
+
+<Anatomy />
+
+Tabs consist of a tablist with one or more visually separated tabs. Each tab has associated content and only the selected tab's content is shown.
```

Comment:
I noticed that in `Features` we say `panels content`, in the daigram it's `Tab View` and here it is `tabs content`. All similar and all terms we use, but wondering if we should be more precise.

Reply:
I've cleaned up the anatomy to say panel
this particular sentence though i think makes the most sense the way it is written, each tab has content that is shown within the panel, and only the selected tab's content is shown